src/main/java/com/minecolonies/entity/pathfinding/PathJobFindTree.java

```diff
@@ -35,9 +35,7 @@ public PathJobFindTree(World world, @NotNull BlockPos start, BlockPos home, int
     public static class TreePathResult extends PathResult
     {
         public BlockPos treeLocation;
-    }
-
-    @NotNull
+    }    @NotNull
     @Override
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Annotation 'Override' have incorrect indentation level 4, expected level should be 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck)